pkg/discovery/dtofactory/property/pod_properties.go

```diff
@@ -53,12 +54,23 @@ func BuildPodProperties(pod *api.Pod) []*proto.EntityDTO_EntityProperty {
 	}
 
 	for _, toleration := range pod.Spec.Tolerations {
-		tagNamePropertyName := toleration.Key
-		tagNamePropertyValue := string(toleration.Operator)
-		if toleration.Value != "" {
-			tagNamePropertyValue += " " + toleration.Value
+		tagNamePropertyName := TolerationPropertyNamePrefix
```

Comment:
Here you mean the same as above the "ToString()" method, right?

Reply:
Apologies, I meant the `String()` method on `tolerations`.
`taint` has a `ToString()` method which emits the concise string representation of `taint`.  I thought its merit in representing the same in the form its expected in k8s:
```
// taint.ToString() converts taint struct to string in format '<key>=<value>:<effect>', '<key>=<value>:', '<key>:<effect>', or '<key>'.

Example: output of describe nodes
Taints:   node.kubernetes.io/unschedulable:NoSchedule
```
`taint` has a `String()` method also which is a much longer representation. 
`toleration` does not have a `ToString()` but only a `String()` which is a default longer representation. I am not sure if this is very useful, that is why I said `explore`.. :-)

Reply:
@irfanurrehman A very good point on retaining the same string representation as in k8s.  I agree about its merit too. Let me bring it up tomorrow in the team standup to see how everybody thinks.

Reply:
@irfanurrehman After thinking about this more, I'm leaning to leave it as is because there isn't an equivalent string representation on "toleration".  Also, my current breakdown is very close to the taint string representation (broken down at the ":"). I think it should be similar enough.